ext/microprofile/mp-rest-client/src/main/java/org/glassfish/jersey/microprofile/restclient/RestClientProducer.java

```diff
@@ -334,6 +334,9 @@ private InputStream locationToStream(String location) throws IOException {
             if (null == result) {
```

Comment:
Can you fix `from teh same classloader` at 332?

Reply:
Fixed